lib/milc_interface.cpp

```diff
@@ -83,6 +83,7 @@ template <bool start> void inline qudamilc_called(const char *func, QudaVerbosit
     } else {
       printfQuda("QUDA_MILC_INTERFACE: %s (return) \n", func);
     }
+    fflush(stdout);
```

Comment:
The `fflush` is redundant here as `printfQuda` include a call to `fflush`

Reply:
addressed in 61eb8fb2063699b3833a2bd1abcf361a3ca12803